src/robusta/core/sinks/sink_base_params.py

```diff
@@ -91,12 +91,21 @@ def validate_exactly_one_defined(cls, values: Dict):
 class GroupingParams(BaseModel):
     group_by: GroupingAttributeSelectorListT = ["cluster"]
     interval: int = 15*60  # in seconds
+    aligned: bool = False
     notification_mode: Optional[NotificationModeParams]
 
     @root_validator
-    def validate_notification_mode(cls, values: Dict):
-        if values is None:
-            return {"summary": SummaryNotificationModeParams()}
+    def validate_interval_alignment(cls, values: Dict):
+        if values["aligned"]:
```

Comment:
what if values is none? wont this throw? i would handle that case first

Reply:
It's marked as required, so only possible values should be `True` and `False`.